### DISCUSSION TITLE
Function "/". This Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by Clydinite** [vercel/community#2550] [Help]
### Summary

![image](https://github.com/vercel/community/assets/116179896/a5ff8ad0-39a8-4bf2-a7b2-c0b0887c82d3)
Serverless function crashes every time I pushed something... Is there any way of fixing that, by the way, I'm using SvelteKit.

https://codetogetherashs.vercel.app/

**Comment by amyegan** [comment]
Since a 500 error is typically either [FUNCTION_INVOCATION_FAILED](https://vercel.com/docs/errors/application-error-codes#function_invocation_failed) or [EDGE_FUNCTION_INVOCATION_FAILED](https://vercel.com/docs/errors/application-error-codes#edge_function_invocation_failed), the most likely cause is an an unhandled exception in the app. More info will show if you click one of these error rows. You can log more details and handle the error by wrapping your function code in a [try/catch](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) statement.

**Comment by Clydinite** [comment]
![image](https://github.com/vercel/community/assets/116179896/9f6f8180-2b2b-446e-985b-da21bbed7787)
But I deployed the code as a static site. There shouldn't be any function calls. Also, the `GET / ` error message is beyond my understandings, is there any ways to solve it? Thanks!

**Reply by amyegan** [reply]
If you are able to share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I can take a closer look
